src/components/core/Heading/Heading.md

```diff
@@ -1,6 +1,6 @@
 ```jsx
-<Heading size="xxlarge" is="h1">Heading 1</Heading>
-<Heading size="xlarge" is="h2">Heading 2</Heading>
-<Heading size="large" is="h3">Heading 3</Heading>
+<Heading size="xxl" is="h1">Heading 1</Heading>
```

Comment:
I see now. I think I still would prefer a styled-system approach with a single typographic scale + DSL props that would map to it. For headings I think it could be `level` that by default also would set the appropriate html tag. Thoughts?

Reply:
Yeah, sounds good! But you'll be able overwrite a tag with `is` prop?